server/server/search/tests/test_instant_search.py

```diff
@@ -120,4 +124,52 @@ def test_normalize_string():
     assert normalize_string('rupa') == 'rupa'
     assert normalize_string('Rūpā') == 'Rupa'
     assert normalize_string('samma') == 'samma'
-    assert normalize_string('sāmma') == 'samma'
\ No newline at end of file
+    assert normalize_string('sāmma') == 'samma'
+
+
+def instant_search(query, languages):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Consider adding assertions for the structure of the response JSON.

Validating the status code and the 'total' field is a good start, but it might be beneficial to also assert the structure of the returned JSON to ensure it meets the expected format, especially for fields that are critical to the functionality being tested.